angular-chart.js

```diff
@@ -362,5 +365,11 @@
       var options = angular.extend({}, Chart.defaults.global, ChartJs.getOptions(type), scope.options);
       return options.responsive;
     }
+
+    function destroyChart(chart, scope) {
+      if(!chart) return;
+      chart.destroy();
+      scope.$emit('destroy', chart);
```

Comment:
Can you update the events section in the README pls? https://github.com/jtblin/angular-chart.js#events